cardano-db-sync/src/Cardano/DbSync/Era/Shelley/Insert.hs

```diff
@@ -251,12 +251,19 @@ insertPoolRegister tracer (EpochNo epoch) txId idx params = do
   rewardId <- insertStakeAddress txId $ Shelley._poolRAcnt params
 
   when (fromIntegral (Shelley.unCoin $ Shelley._poolPledge params) > maxLovelace) $
-    liftIO . logError tracer $
+    liftIO . logWarning tracer $
       mconcat
         [ "Bad pledge amount: ", textShow (Shelley.unCoin $ Shelley._poolPledge params)
         , " > maxLovelace. See https://github.com/input-output-hk/cardano-ledger-specs/issues/1551"
         ]
 
+  when (fromIntegral (Shelley.unCoin $ Shelley._poolCost params) > maxLovelace) $
+    liftIO . logWarning tracer $
+      mconcat
+        [ "Bad fixed cost amount: ", textShow (Shelley.unCoin $ Shelley._poolCost params)
+        , " > maxLovelace. See https://github.com/input-output-hk/cardano-db-sync/issues/351"
+        ]
+
```

Comment:
It is not necessary to have this warning. It is permitted to declare pool costs that could not in practice be satisfied (and there is no line we could draw in advance that separates satisfiable from unsatisfiable values).